vault/identity_store_aliases.go

```diff
@@ -255,6 +251,12 @@ func (i *IdentityStore) handleAliasUpdateCommon() framework.OperationFunc {
 			return nil, err
 		}
 
+		for index, item := range entity.Aliases {
```

Comment:
Same, can you explain what this change is for?

Reply:
All the updates are held by the `alias` object. This is where we reflect those changes in the entity.